src/ResourceManager/AzureBackup/Commands.AzureBackup/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.BackupServices" version="1.0.5-preview" targetFramework="net45" />
-</packages>
+  <package id="Microsoft.WindowsAzure.Management.Scheduler" version="6.0.0" targetFramework="net452" />
```

Comment:
Check this dependency

Reply:
This dependency is required. Going to create an issue for removing it.